wicker/core/datasets.py

```diff
@@ -131,8 +221,8 @@ def __getitem__(self, idx: int) -> Dict[str, Any]:
             self.schema(),
         )
 
-    def _get_parquet_dir_size(self) -> int:
-        """Get the arrow path and find all the files within, count their bytes
+    def _get_parquet_dir_size(self, should_compute_entire_parquet_dir_size: bool = False) -> int:
```

Comment:
@pickles-bread-and-butter if the argument `should_compute_entire_parquet_dir_size` isn't used in the function any longer, can we remove it?

Reply:
Oh sorry, I thought I had gotten it